packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -72,6 +76,21 @@ const Dashboard = observer(() => {
     };
     const active_hash_tab = GetHashedValue(active_tab);
 
+    const connectionDistrupted = () => {
```

Comment:
There's a typo here 

```suggestion
    const connectionDisrupted = () => {
```

Also the name of the function is not exactly what it is doing. You are checking whether the connection is disrupted and then doing some action. We. can instead use something like: `checkAndHandleConnection`

Reply:
done ✅